utils.py

```diff
@@ -110,11 +110,18 @@ def import_article_metadata(request, reader):
             articles[article_id] = article
 
         # author import
+        *author_fields, is_corporate = author_fields
+        article = articles[article_id]
+        if is_corporate == "Y":
```

Comment:
Can almost guarantee someone will put lowercase y in, could we check for `is_corporate.upper()`?